hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/DBCheckpointServlet.java

```diff
@@ -276,6 +272,16 @@ private void generateSnapshotCheckpoint(HttpServletRequest request,
     }
   }
 
+  protected static Set<String> fetchSstFilesReceived(String[] sstParam) {
```

Comment:
Suggest to rename the method to better describe the intent.
```suggestion
  protected static Set<String> extractSstFilesToExclude(String[] sstParam) {
```